packages/@aws-cdk/core/lib/custom-resource-provider-handler.ts

```diff
@@ -0,0 +1,139 @@
+// tslint:disable: no-console
+import * as https from 'https';
+import * as url from 'url';
+
```

Comment:
Can we sub-folder these files under `lib/custom-resource/`, so the different parts of core are organized.

Reply:
sure

Reply:
I'll move to `custom-resource-provider` (which is not the same as `custom-resource.ts`).

Reply:
Done